sources/functions/short

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+#please make sure to give this full paths
+function rm_if_exists () {
+    path="$1"
+    if [[ -e "$path" ]]; then
+      rm -rf "$path"
+    fi
+}
+
```

Comment:
I wonder whether the filename for this is appropriate. I chose `short` as I intended this to store other small shortcuts.